<ui version="4.0" >
 <class>CGexEmailMainwindow_base</class>
 <widget class="QMainWindow" name="CGexEmailMainwindow_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gex Email Dispatcher</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="gex_email.qrc" >
    <normaloff>:/gex_email/images/resources/galaxy_icon.png</normaloff>:/gex_email/images/resources/galaxy_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>9</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listMainSelection" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="movement" >
         <enum>QListView::Static</enum>
        </property>
        <property name="isWrapping" stdset="0" >
         <bool>true</bool>
        </property>
        <property name="viewMode" >
         <enum>QListView::IconMode</enum>
        </property>
        <property name="uniformItemSizes" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="widgetStack" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageGeneral" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupWidgetGeneral" >
            <property name="title" >
             <string>General</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3" >
             <item row="0" column="0" >
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item row="1" column="1" >
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="editEmailDirectory" >
                   <property name="enabled" >
                    <bool>false</bool>
                   </property>
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonBrowseEmailDirectory" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize" >
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize" >
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text" >
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="1" >
                <layout class="QHBoxLayout" >
                 <property name="spacing" >
                  <number>6</number>
                 </property>
                 <property name="margin" >
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QSpinBox" name="spinBoxHours" >
                   <property name="maximum" >
                    <number>999</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelHours" >
                   <property name="text" >
                    <string>h</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBoxMinutes" >
                   <property name="maximum" >
                    <number>59</number>
                   </property>
                   <property name="value" >
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelMinutes" >
                   <property name="text" >
                    <string>min</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBoxSeconds" >
                   <property name="maximum" >
                    <number>59</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelSeconds" >
                   <property name="text" >
                    <string>sec</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="1" column="0" >
                <widget class="QLabel" name="labelEmailDirectory" >
                 <property name="text" >
                  <string>Email directory:</string>
                 </property>
                 <property name="buddy" >
                  <cstring>editEmailDirectory</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" >
                <widget class="QLabel" name="labelPeriodicity" >
                 <property name="text" >
                  <string>Send email every:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0" >
              <widget class="QGroupBox" name="groupBoxAdmin" >
               <property name="title" >
                <string>Admin</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2" >
                <item row="0" column="0" >
                 <layout class="QGridLayout" name="gridLayoutAdmin" >
                  <item row="0" column="0" >
                   <widget class="QLabel" name="labelNbLogFiles" >
                    <property name="text" >
                     <string>Nb of log files to keep:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1" >
                   <widget class="QSpinBox" name="spinDaysToKeepLogFiles" >
                    <property name="minimum" >
                     <number>1</number>
                    </property>
                    <property name="maximum" >
                     <number>999</number>
                    </property>
                    <property name="value" >
                     <number>1</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2" >
                   <spacer name="horizontalSpacer" >
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0" >
                     <size>
                      <width>208</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0" >
                 <widget class="QCheckBox" name="checkBoxFullLog" >
                  <property name="text" >
                   <string>Full log (log sent emails)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0" >
              <spacer>
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageLog" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupWidgetLog" >
            <property name="title" >
             <string>Log</string>
            </property>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>9</number>
             </property>
             <item>
              <widget class="QTreeWidget" name="treeWidgetLog" >
               <column>
                <property name="text" >
                 <string>1</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <property name="spacing" >
                <number>6</number>
               </property>
               <property name="margin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="buttonClearLog" >
                 <property name="text" >
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageHelp" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupWidgetHelp" >
            <property name="title" >
             <string>Help</string>
            </property>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>9</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="textBrowserHelp" >
               <property name="html" >
                <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">Under construction ...&lt;/p>&lt;/body>&lt;/html></string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameButtons" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonSave" >
           <property name="text" >
            <string>Save Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonStartStop" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonHelp" >
           <property name="text" >
            <string>Help</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonExit" >
           <property name="text" >
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupStatus" >
      <property name="title" >
       <string>Status</string>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="labelStatus" >
         <property name="text" >
          <string>Ready</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <tabstops>
  <tabstop>listMainSelection</tabstop>
  <tabstop>spinBoxHours</tabstop>
  <tabstop>spinBoxMinutes</tabstop>
  <tabstop>spinBoxSeconds</tabstop>
  <tabstop>editEmailDirectory</tabstop>
  <tabstop>buttonBrowseEmailDirectory</tabstop>
  <tabstop>treeWidgetLog</tabstop>
  <tabstop>buttonClearLog</tabstop>
  <tabstop>textBrowserHelp</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonStartStop</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonExit</tabstop>
 </tabstops>
 <resources>
  <include location="gex_email.qrc" />
 </resources>
 <connections/>
</ui>
